main.go

```diff
@@ -24,44 +24,48 @@ import (
 	"github.com/chromedp/chromedp"
 )
 
-var (
-	cpuProfile      *string
-	coverageProfile *string
-)
-
 func main() {
-	// NOTE: Since `os.Exit` will cause the process to exit, this defer
-	// must be at the bottom of the defer stack to allow all other defer calls to
-	// be called first.
-	exitCode := 0
+	exitCode := run(os.Args, os.Stderr, flag.CommandLine)
+	os.Exit(exitCode)
+}
+
+func run(args []string, errOutput io.Writer, flagSet *flag.FlagSet) (exitCode int) {
```

Comment:
This function should arguably just return errors which always result in `exitCode = 1` inside `main()`. I left it like this to avoid a larger refactor, but I'd be happy to go further if you agree.

Reply:
Done in e0a1f8a